File: card.py
from details import heirarchies, numbers, suits, gameTypes


class Card:
    def __init__(self, suit, number):
        """Making Card object with suit and Number"""
        self.suit = suits[suit]
        self.number = numbers[number]


card1=Card("spade",2)
print(card1.number)

File: JS-logic/card.js
const { heirarchies, suits, numbers, gameTypes } = require("./details")

class Card {
  constructor(suit, number) {
    this.suit = suits[suit];
    this.number = numbers[number]
  }
}

let c1 = new Card();

module.exports = {
  Card,c1
}

// let card1 = new Card("spade", 2);
// console.log(card1)